doc/guide/cli_quickstart.hpp

```diff
@@ -108,7 +108,7 @@ The example above has only shown a little bit of the functionality of mlpack.
 Lots of other commands are available with different functionality.  A full list
 of commands and full documentation for each can be found on the following page:
 
- - <a href="https://mlpack.org/doc/mlpack-git/cli_documentation.html">IO documentation</a>
+ - <a href="https://mlpack.org/doc/mlpack-git/cli_documentation.html">CLI documentation</a>
```

Comment:
Ahhh, thank you.  Actually do you want to submit this one as a patch batch to the mlpack repository?